blaze/expr/tests/test_collections.py

```diff
@@ -23,7 +24,8 @@ def test_transform():
     expr = transform(t, z=t.x + t.y)
     assert expr.fields == ['x', 'y', 'z']
 
-    assert (t.x + t.y) in expr._subterms()
+    assert builtins.any((t.x + t.y).isidentical(node)
```

Comment:
Ironically this would have worked fine if we had star-imported from `blaze` rather than `blaze.expr`.